src/extlibs/hrvo/linear_velocity_agent.cpp

```diff
@@ -30,39 +30,41 @@ void LinearVelocityAgent::computeNewVelocity()
         // Calculate the maximum velocity towards the preferred velocity, given the
         // acceleration constraint
         new_velocity_ =
-            velocity_ + (max_accel_ * simulator_->getTimeStep()) * (dv / abs(dv));
+            velocity_ + (max_accel_ * simulator_->getTimeStep()) * (dv / dv.length());
```

Comment:
pretty sure this works
```suggestion
            velocity_ + (max_accel_ * simulator_->getTimeStep()) * (dv.normalize());
```